### PR TITLE
WIP: Behavior report

### PR DESCRIPTION
**Pull request by peterasujan**

### CODE REVIEW

**Comment by peterasujan** [comment]
@dylandaniels mind taking a quick lookthrough when you get a chance?

File: doc/source/report/behavior.rst [hunk 1]
```diff
@@ -50,165 +50,70 @@ to capture the behavioral profile:
...
+In this code, we see a few important features. First, we create
+behavior trees using the ``compute_tree`` function, and demonstrate
+their pretty-printing functionality.  Then, we merge lists of trees
+into one larger tree of lists, which we can then summarize or turn
+into a Pandas data frame. Finally, we show how to plot the results
+using ``matplotlib``.
 
-- the observations of location for each mice, $(x, y, t)$ with $t$ small.
-- the aggregated time binned features about each event and its intensity.
+.. literalinclude:: report/plots/behavior_example.py
```
Comment (jarrodmillman) [inline doc/source/report/behavior.rst]: I think you can do something like:

```
To also include the source code for the plot in the document, pass the include-source parameter:

.. plot:: pyplots/ellipses.py
   :include-source:
```

I am using modified code from matplotlib, so most of this should be true:
http://matplotlib.org/sampledoc/extensions.html

I submitted the changes upstream as well:
https://github.com/matplotlib/matplotlib/pull/6327

**Comment by dylandaniels** [comment]
@peterasujan @shamindras looks good to me! 

Is there a way I can test the change Jarrod suggested to the Sphinx plot+source generation command locally? Not sure how to do this.

**Comment by peterasujan** [comment]
Good call @jarrodmillman, I just made that change. I believe things are ready to merge.

**Pull request merged**